docs/specifications/tests/Zone-TP/zone11.md

```diff
@@ -0,0 +1,153 @@
+# ZONE11: SPF policy validation
+
+## Test case identifier
+**Zone11**
+
+## Table of contents
+
+* [Objective](#objective)
+* [Scope](#scope)
+* [Inputs](#inputs)
+* [Summary](#summary)
+* [Test procedure](#test-procedure)
+* [Outcome(s)](#outcomes)
+* [Special procedural requirements](#special-procedural-requirements)
+* [Intercase dependencies](#intercase-dependencies)
+* [Terminology](#terminology)
+
+## Objective
+
+Sender Policy Framework (SPF), described in [RFC 7208][RFC7208], is a mechanism
+allowing domain name owners to specify which hosts are allowed to send mail
+claiming to be from that domain. It is implemented by means of TXT records in a
+structured format.
+
+This test looks up SPF records in the domain to be tested. It ensures that
+there is at most one such record and, if present, ensures it is syntactically
+valid and that its evaluation does not require more than 10 DNS lookups.
+
+## Scope
+
+It is assumed that *Child Zone* has been tested and reported by [Basic04]. This
+test case will just ignore non-responsive name servers or name servers not
+giving a correct DNS response for an authoritative name server.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+## Summary
+
+Message Tag outputted           | Level   | Arguments            | Description of when message tag is outputted
+:-------------------------------|:--------|:---------------------|:--------------------------------------------
+SPF1_FOUND_AND_VALID            | INFO    |                      | The domain publishes a valid SPF version 1 policy.
+SPF1_LOOKUP_LIMIT_EXCEEDED      | ERROR   |                      | Evaluating the published SPF version 1 policy would require more than 10 DNS lookups.
+SPF1_MORE_THAN_ONE_SAME_MODIFIER | ERROR  | modifier             | The published SPF version 1 policy contains the same modifier (redirect or exp) more than once.
```

Comment:
I think one PR is enough unless there are multiple updates of the argument list, then I guess an issue would be fine so that we do not forget it.

I guess that the second question we should ask if a new argument name is needed, or if a generic name could be used. If a specific one should be used, should the argument name be `spf_modifier` or `spf_mod` (to keep it shorter).

Reply:
Oh sorry, I haven’t pushed a new version of the PR yet. I was also going to change the argument to `spf_modifier`.